src/test/resources/pages/studentCourseJoinConfirmationHTML.html

```diff
@@ -273,13 +273,13 @@
 	    <div class="panel-body">
 	        <p>
 	            You are currently logged in as <span><strong>{$test.student1}</strong></span>. 
-	            <br />If this is not you please <a href="{*}">log out</a> and re-login using your own Google account.
+	            <br />If this is not you please <a href="/_ah/logout?continue=%2Fpage%2FstudentCourseJoinAuthenticated%3Fkey%3D{*}%26next%3D%2Fpage%2FstudentHomePage">log out</a> and re-login using your own Google account.
```

Comment:
If only we could fix godmode to not incorrectly regenerate these :\ It's hard to notice because the person who made the PR will have these pass.